pkg/assetfetcher/config.go

```diff
@@ -39,5 +41,6 @@ func (*AssetFetcherConfigComponent) New(_ context.Context, c *AssetFetcherConfig
 	return &NexposeAssetFetcher{
 		Host:     host,
 		PageSize: c.PageSize,
+		StatFn:   runhttp.StatFromContext,
```

Comment:
Nit: we've been avoiding using the runhttp library directly in code, trying to stick to using interfaces for loggers/staters so we can replace them more easily. The only place in code that references the specific runhttp package is in main.go. This also means we can generate mock loggers/staters for our tests. See some examples of this pattern from asset-inventory-api:
* https://github.com/asecurityteam/asset-inventory-api/blob/master/main.go#L27
* https://github.com/asecurityteam/asset-inventory-api/blob/master/pkg/domain/alias.go
* https://github.com/asecurityteam/asset-inventory-api/blob/master/pkg/handlers/v1/cloud_insert.go#L32
* https://github.com/asecurityteam/asset-inventory-api/blob/master/pkg/handlers/v1/stub_logger_test.go

Reply:
Here's a much better discussion about why to do this: https://github.com/asecurityteam/nexpose-asset-attributor/pull/1#discussion_r278641948